rdrf/rdrf/exporter.py

```diff
@@ -496,6 +496,7 @@ def _get_context_form_groups(self):
             cfg_dict["forms"] = []
             for form in cfg.forms:
                 cfg_dict["forms"].append(form.name)
+            cfg_dict["ordering"] = cfg.ordering
```

Comment:
hm, I was sure that I did and that I tested it, but lost the change somehow.
Thanks for picking it up :)